### ISSUE TITLE
DZI file not being displayed.  http 404 unable to open object.

### ISSUE THREAD

**Issue by apricea4** [openseadragon/openseadragon#1261]
Here is my relevant JS,

var viewer = OpenSeadragon({
                    id: "seaDragonViewer",
                    prefixUrl: "openseadragon/images/",
                    tileSources: "dzi/realSlide/imageSlide.dzi",

and this is my file structure,

![capture](https://user-images.githubusercontent.com/22583746/28343568-d988bc32-6bd2-11e7-9e70-c46285aed41f.PNG)

I used vips to convert a large tiff to dzi.  

Has anyone else used vips successfully?

**Comment by iangilman** [comment]
Looks good, but the `dzc_output_files` in the realSlide folder should be named `imageSlide_files` (it needs to match the DZI file name).

**Comment by apricea4** [comment]
Thank you for your help.  I made the change and now it has changed to "404 attempting to open tile source"  I have tried viewing it by converting it to the zoomify format and displaying the plain .tiff and specifying type: image or zoomify, but it will not display.  The original tiff is ~200 MB, has size ever been an issue for you?

**Comment by iangilman** [comment]
Sounds like you just need to get your paths right somehow. Can you show your full file structure including the HTML and JavaScript files? 

Original image size is not a problem (assuming you've converted to something like Zoomify or DZI). This OpenSeadragon image is 365 gigapixels (in DZI format, I believe): 

http://www.in2white.com/

You can't use type: image for .tiff files, but you can certainly convert tiff files to DZI or zoomify and use that. 

At any rate, since you already have a DZI, we should just figure out what's wrong with the URLs you're using.

**Comment by apricea4** [comment]
Sure thing, I have made some changes to the file structure from last time but realslide is still parallel to the html file.  
<img width="131" alt="filestructure1" src="https://user-images.githubusercontent.com/22583746/28438386-ce7a5a22-6d52-11e7-8c1b-39756e9f36d6.PNG">

index.html is where the viewer is located.  

Thank you.

**Comment by iangilman** [comment]
Very strange... your folder structure and code look perfect. The only thing I can think of to try is changing `outputFiles.dzi` to `outputFiles.xml`; some webservers prefer the more familiar filetype (OSD doesn't care). 

I'm not sure what else to suggest... you could send me the project so I can try it on my machine (you can send me a link on Gitter or via ian@iangilman.com if you don't want to share it publicly).

**Comment by apricea4** [comment]
Thank you very much for your help @iangilman.  I swear I tried switching the extension to .xml before asking this question, but that is what did it for me.  From my reading, I was under the impression they were interchangeable and Microsoft did not care which one was used.  I am using VS 2017 and I just renamed the extension to .xml from .dzi in the solution explorer and it was displayed properly.

Cheers
